tests/unit/data/login/socket_localhost:22420.log

```diff
@@ -0,0 +1,86 @@
+(dp0
```

Comment:
This whole file and any other derived files shouldn't be checked into SCM.

Reply:
This is needed to mock the server traffic in unit tests. The ones in the `data/search` folder are not utilized yet though, so I took them out.